envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -377,6 +378,22 @@ class DynamicForwardProxyProperties {
     var connectionTimeout = Duration.ofSeconds(1)
 }
 
+class CompressionProperties {
+    var gzip = GzipProperties()
+    var brotli = BrotliProperties()
```

Comment:
can they be both enabled? or only one?

Reply:
they can be both declared in configuration, but the one with "chooseFirst=true" setting will be chosen